website/docs/components/time/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Time
+description: The Time component is used to display dates and time in a consistent format.
+caption: Displays dates and times in a consistent format.
+related: ['components/form/text-input']
```

Comment:
Suggestion: another related component could be Hds::Text

Reply:
I'm not sure that Text::Input is related actually myself. Text may be more related though so thanks for the suggestion.

Reply:
@andgen404 Would it be ok if I remove Text Input and add Text instead as being related? The Time component doesn't allow for any type of input so to me doesn't seem related to the Text Input.

Reply:
I went ahead and made this update for now. We can change again if needed